addons/website_event_track/views/event_track_views.xml

```diff
@@ -302,6 +302,37 @@
             groups="base.group_no_one"/>
 
         <!-- EVENT.SPONSOR VIEWS -->
+        <record id="event_sponsor_view_kanban" model="ir.ui.view">
```

Comment:
I would vote to move all sponsors views in their own file (to be done in a first commit, then adding new views related to this task).